### ISSUE TITLE
Import cause crash

### ISSUE THREAD

**Issue by Ernold** [pyfa-org/Pyfa#1472]
## Bug Report
[pyfa error report .txt](https://github.com/pyfa-org/Pyfa/files/1794911/pyfa.error.report.txt)

### Expected behavior:
Import single fitting file form hard drive, Pyfa has started normally, this was first command

### Actual behavior:
Pyfa crash  
and cause Win 10 64bit desktop PC freeze

### Detailed steps to reproduce:
I reinstalled Pyfa

### Release or development git branch? Please note the release version or commit hash:
pyfa-v2.0.0b4-win

### Operating system and version (eg: Windows 10, OS X 10.9, OS X 10.11, Ubuntu 16.10):
Windows 10 64bit 1709 release ok

**Comment by blitzmann** [comment]
You sure this was due to an import? Looks like it was caused by deleting a fit (which is a known issue)

**Comment by Ernold** [comment]
I used the import command, selected single Widow fit from ssd , imported it and…. nothing fit screen was labelled as that Widow fit but nothing opened and after a while ”No Response” and pc freeze when I tried to shut down Pyfa.

Eero

Lähetetty Windows 10:n Sähköpostista

Lähettäjä: Ryan Holmes
Lähetetty: torstai 8. maaliskuuta 2018 23.44
Vastaanottaja: pyfa-org/Pyfa
Kopio: Ernold; Author
Aihe: Re: [pyfa-org/Pyfa] Import cause crash (#1472)

You sure this was due to an import? Looks like it was caused by deleting a fit (which is a known issue)
